conf/config.sample.yaml

```diff
@@ -1,6 +1,12 @@
 server:
   port: 8080
   rootpath: "/" # the path at which the router is mounted
+  domain: "127.0.0.1"
+auth:
+  loginModuleURL: "http://127.0.0.1:8000"
+  clientID: "1"
+  clientSecret: "tzxsLyFtJiGnmD6sjZMqSEidVpVsL3hEoSxIXCpI"
+  callbackURL: "http://127.0.0.1:8080/auth/login-callback"
```

Comment:
'callbackURL' is a server protocol (which we don't have in the config) + domain + port + callbackPath (can be added to the config). Should I remove the 'Auth.CallbackURL' and add 'Server.Protocol' and 'Auth.CallbackPath' instead?

Reply:
But we can assume the callback is on the same server, so both the domain, protocol and port should not be mentioned. I assume we can get from the request all these (protocol, domain, port). 
However, it should be forbidden to use HTTP (<> HTTPS) for the auth callback. Do browsers allow redirect between HTTP and HTTPS? If not, it is already this service (/auth/login) which should not be possible in HTTP.

Reply:
Unfortunatelly, we cannot get the protocol from the request until we don't know how the load balancer will inform us if it is https or http (and we even cannot get the port from the request because the app can listen on a port that differs from the load balancer's one). For the same reason, it is not obvious how to forbid HTTP requests.

Reply:
Probably we should forbid HTTP requests to auth/login on the load balancer level.